data/rulesets.ts

```diff
@@ -826,6 +826,15 @@ export const Formats: {[k: string]: FormatData} = {
 			this.add('rule', 'Switch Priority Clause Mod: Faster Pokémon switch first');
 		},
 	},
+	desyncclausemod: {
+		effectType: 'Rule',
+		name: 'Desync Clause Mod',
+		desc: 'In the event a move would cause a desync, the move will fail instead. This rule covers Psywave and Counter.',
+		// The effect itself is implemented in gen1/moves.ts. 
+		onBegin() {
+			this.add('rule', 'In the event a move would cause a desync, the move will fail instead.');
```

Comment:
Design principle 1 is in full effect, especially in rule descriptions where the length limit is _very_ low.

```suggestion
			this.add('rule', 'Desync Clause Mod: Desyncs become move failure.');
```